ocean_lib/ocean/ocean_assets.py

```diff
@@ -163,6 +166,13 @@ def create(
             logger.error(msg)
             raise ValueError(msg)
 
+        urls = [item["url"] for item in metadata["main"]["files"]]
+        for url in urls:
+            if not DataServiceProvider.check_single_file_info(url, provider_uri):
+                msg = f"Some files can not be accessed: {url}."
```

Comment:
It may be a file, or it may be another service. And, you only know right now that this _single_ service can't be accessed, not >1. Perhaps better phrasing: "The following service could not be accessed: {url}."

Reply:
I just checked. It now says "The URL of this file can not be accessed: {url}".

That fixes my second concern. 

My first concern still needs addressing: It may be a file, or it may be another service.